recipes/fastjet/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "fastjet" %}
+{% set version = "3.4.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastjet-{{ version }}.tar.gz
+  sha256: 86dea05ce7a030db9fe14e5bf1c0bc46351b6e439f9b9ef6b14a55ceefcc7b7a
+  patches:
+    - 0001-Don-t-overwrite-external-CXXFLAGS-and-LDFLAGS.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  missing_dso_whitelist:
+    - $RPATH/libfastjet.so.0
+    - $RPATH/libfastjettools.so.0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - libtool
+    - automake
+    - autoconf
+    - swig
```

```suggestion
    - swig
    - patch
    - gmp  # In 'build' over 'host' as avoids 'ld: cannot find -lgmp: No such file or directory' error
```

Though the `patch` executable is already in the Docker image that conda-forge uses for builds, it is directly used in the `setup.py` and so should be given here too.

The fact that we need `gmp` in `build` and _not_ `host` to avoid a

```
ld: cannot find -lgmp: No such file or directory
```

error is confusing to me though, as we're using the shared object information from `libgmp` and not the header files, so I'm unclear as to why this needs to be under `build` and so `BUILD_PREFIX`, when I would have guessed `host` and `PREFIX` to make more sense. Especially given the

> For example, in order for a recipe to be "cross-capable", shared libraries requirements must be listed in the host section, rather than the build section, so that the shared libraries that get linked are ones for the target platform, rather than the native build platform.

bit from https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#host.